### ISSUE TITLE
Generating evaluation for ActivityNet dataset without score fusion

### ISSUE THREAD

**Issue by vc-30** [happyharrycn/actionformer_release#34]
Hi @happyharrycn,

Thank you for the great work. 

I am trying to run the inference code for ActivityNet dataset. Currently the pretrained model supports score fusion due to which labels are coming from the `cuhk_val_simp_share.json`  file. I want to try generating numbers without score fusion, and I think the pretrained model currently don't support that configuration.

I tried making the changes in `./configs/anet_tsp.yaml` by below changes:

```diff
@@ -51,9 +51,9 @@ test_cfg: {
   max_seg_num: 100,
   min_score: 0.001,
   # score fusion
-  multiclass_nms: False,
+  multiclass_nms: True,
   nms_sigma : 0.75,
-  ext_score_file: ./data/anet_1.3/annotations/cuhk_val_simp_share.json,
+  #ext_score_file: ./data/anet_1.3/annotations/cuhk_val_simp_share.json,
   duration_thresh: 0.1,
 }
```

With the above changes I got 0 mAP, probably because the `num_classes` is configured as 1 currently.
Also, I could reproduce the number for THUMOS dataset where the score fusion is disabled and `num_classes` is set to 20 which are the number of action classes for the dataset.

**Comment by vc-30** [comment]
Thank you for your response.

In that case is it possible to share the model that used R(2+1)D encoder trained on Thumos dataset without score fusion.

I am trying to do the inference on custom videos. 

I was also looking at converting videos to I3D features using [CMCS](https://github.com/Finspire13/CMCS-Temporal-Action-Localization) and [CMCS-I3D](https://github.com/Finspire13/pytorch-i3d-feature-extraction) is bit unclear (converting video to rgb & flow or just rgb images, feed to the extract_features.py)

**Comment by happyharrycn** [comment]
We can look into the release of a pretrained model using R(2+1)D features, yet it will be of lower priority on our list. It would be straightforward to train such a model on your own using the features [here](https://github.com/HumamAlwassel/TSP). The training will take less than half an hour with a modern GPU.

For I3D features, we used two-stream I3D models. Specifically, the videos were resized and downsampled. RGB frames and optical flows were extracted and fed into the RGB and flow stream, respectively. The resulting feature vectors were concatenated for each clip.

**Comment by vc-30** [comment]
ok thanks for the inputs. I can try the retraining part for R(2+1)D features for both actually THUMOS and ActivityNet.

For I3D features, I can try going through the given [link](https://github.com/happyharrycn/actionformer_release/issues/34#issuecomment-1145379385), and probably open another issue in case of any queries.

I will close the issue for now, and would look out for the pre-trained models in case you share it.

Thanks!
